odk1-src/form-functions.rst

```diff
@@ -0,0 +1,502 @@
+Form Functions
+================
+
+:ref:`expressions` in :ref:`calculations`, :ref:`constraints <constraints>`, and :ref:`relevants <relevants>`
+can contain `XPath functions`_.
+
+.. _XPath functions: https://opendatakit.github.io/xforms-spec/#xpath-functions
+
+.. admonition:: Helpful terms
+
+  .. glossary::
+    :sorted:
+
+    nodeset
+
+      A collection of response values. 
+
+      Outside a :ref:`repeat group <repeats>`, 
+      :ref:`referring to a question by name <variables>`
+      will return a nodeset containing all the responses to that question.
+
+.. contents::
+  :local:   
+
+
+  
+Form data 
+----------------
+
+Accessing previous responses
```

Comment:
Feels like most of this has to do with selects, so why not have a selects section, then have just a repeats section (even if it has one item). Then could put the mathy bits in the math section under their own subsection, move position and itext to utility.